ci-tools/release/build_release.sh

```diff
@@ -12,29 +12,16 @@ fi
 
 WORKSPACE_DIR="release/workspace"
 release_scripts_path=$(dirname "$0")
-# Generate Release Folder
-rm -rf release
-mkdir -p $WORKSPACE_DIR
 
-# Generate ROM and Image Bundle Binary
-cargo run --manifest-path=builder/Cargo.toml --bin image -- --rom-no-log $WORKSPACE_DIR/caliptra-rom.bin --fw $WORKSPACE_DIR/image-bundle.bin
-# Copy ROM ELF
-cp -a target/riscv32imc-unknown-none-elf/firmware/caliptra-rom $WORKSPACE_DIR/caliptra-rom.elf
```

Comment:
I'm not sure I understand why the things in this file need to be moved to the action definition.

I think the desire with these scripts is that someone can run them locally to build the expected release artifacts.

Reply:
I suppose that's valid. The intention was to build the ROM with the chosen tag first, before checking out HEAD and building some more. I wanted to avoid an actual `git checkout` command here, which would mean calling this has side effects.

Other solutions include passing git commit hashes to this script, and running `git checkout` here (if the CI is okay with it, and people won't mind the side effects that calling it will have to their working directory, it's an option). Alternatively, we could pass arguments to the script to determine which build stage to run: ROM, FMC/RT, or both. Then the CI/developer is responsible for making any changes in between.